Common/src/ArmoniK.Core.Common.csproj

```diff
@@ -26,7 +26,7 @@
 
 
   <ItemGroup>
-    <PackageReference Include="ArmoniK.Api.Core" Version="3.10.0" />
+    <PackageReference Include="ArmoniK.Api.Core" Version="3.11.0-feataddtaskoptionstosession.23.58da3e7" />
```

Comment:
Can this API version go on main?

Reply:
Not yet, we need to merge the new Api first